src/hist/basehist.py

```diff
@@ -401,20 +403,45 @@ def plot2d_full(
 
         return hist.plot.plot2d_full(self, ax_dict=ax_dict, **kwargs)
 
+    def plot_ratio(
+        self,
+        other: Union[T, Callable[[np.ndarray], np.ndarray], str],
```

Comment:
```suggestion
        other: Union["hist.BaseHist", Callable[[np.ndarray], np.ndarray], str],
```
You don't need a TypeVar here, you mostly use that to pin something going out to match what's going in. Strings are fine (and this is part of why 3.7 has annotations as strings).

Reply:
SGTM. I have no also realized (doh!) that you can quote single types and not the whole collection of types.